docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -166,6 +167,7 @@ To switch the Deckhouse cluster to using a third-party registry, follow these st
 
 * Update the `image` field in the `d8-system/deckhouse` deployment to contain the address of the Deckhouse image in the third-party-registry;
 * Edit the `d8-system/deckhouse-registry` secret (note that all parameters are Base64-encoded):
+  * To correctly write values to secrets without use: `echo -n 'value' | base64`;
```

Comment:
```suggestion
  * To correctly write values to secrets without use: `echo -n 'value' | base64`;
```

What about `base64 -w0 <<< secret`  ?  Seems better. Anyway I think it is better to add -w0 parameter to the base64.